src/platform/plugins/shared/workflows_management/public/widgets/workflow_yaml_editor/ui/workflow_yaml_editor.tsx

```diff
@@ -403,6 +429,65 @@ export const WorkflowYAMLEditor = ({
     return () => disposable.dispose();
   }, [isEditorMounted, dispatch]);
 
+  // Subscribe to computed changes and refresh suggestions if visible
+  // Track computedFromYamlString to detect when debounced computation completes
+  const computedFromYamlString = computed?.computedFromYamlString;
+  const computedFromYamlStringRef = useRef<string | undefined>(computedFromYamlString);
+
+  useEffect(() => {
+    if (!isEditorMounted || !editorRef.current) {
+      return;
+    }
+
+    // Only refresh if computedFromYamlString actually changed (computation completed)
+    const hasComputationCompleted =
+      computedFromYamlString !== undefined &&
+      computedFromYamlString !== computedFromYamlStringRef.current;
```

Comment:
I don't think it's necessary to check that, We could do it by only checking the `computed` value is defined and is not an empty object (it's set to `{}` while processing), then we can refresh the widget (if open). 
The new `computedFromYamlString` prop should not be necessary